source/plugin/basics/items.rst

```diff
@@ -0,0 +1,50 @@
+==================
+Working with Items
+==================
+
+Items are represented through an ``ItemStack``. An ``ItemStack`` is an inventory item with information such as the amount of the item in the stack, the type of the item, and extra data such as durability. An item itself is the graphical representation of an ``ItemStack`` as an entity.
```

Comment:
"An item itself..." - please make "Item" capitalized and in an in-line code block: `Item`, as it is a class that you are referring to.